frontend/src/metabase/components/ListField/ListField.tsx

```diff
@@ -60,7 +60,8 @@ export const ListField = ({
   }, [augmentedOptions.length]);
 
   const [filter, setFilter] = useState("");
-  const debouncedFilter = useDebouncedValue(filter, SEARCH_DEBOUNCE_DURATION);
+  const waitTime = useContext(waitTimeContext);
+  const debouncedFilter = useDebouncedValue(filter, waitTime);
```

Comment:
This debounce time was causing flaky Loki tests before because the UI takes time to update.